flow/Makefile

```diff
@@ -178,6 +178,11 @@ ifeq (, $(strip $(NPROC)))
 endif
 export NUM_CORES := $(NPROC)
 
+LSORACLE_PLUGIN ?= oracle
+ifneq ($(USE_LSORACLE),)
```

Comment:
Why not reverse these two lines?  Is there a value in setting LSORACLE_PLUGIN if USE_LSORACLE is unset?

Reply:
My unconscious reason, if I think about it, would be that it's bad form to set a variable inside a conditional block. then you not only have to check later for value, but also presence in scope. i don't know how much that matters.